samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -260,7 +253,7 @@ public async Task<string> AcquireExternalInformationAsync(SKContext context)
         else
         {
             // Create a plan and set it in context for approval.
-            Plan plan = await this._planner.CreatePlanAsync(plannerContext.Variables.Input);
+            Plan plan = await this._planner.CreatePlanAsync(context["userIntent"]);
```

Comment:
Something to consider, feel free to resolve. I wonder if it's better to include both the userIntent and the original Input. I noticed in my testing that `userItent` sometimes loses details of the users ask and including both allows the planner to include as much context as possible.